src/RestServer/Controllers/v1/WalletController.cs

```diff
@@ -0,0 +1,646 @@
+            Guid sessionId)
+        {
+            if (WalletSessions.ContainsKey(sessionId) == false)
+                throw new KeyNotFoundException(sessionId.ToString("n"));
+            var session = WalletSessions[sessionId];
+            var wallet = session.Wallet;
+            session.ResetExpiration();
+            var keys = new List<WalletKeyModel>();
+            foreach (var account in wallet.GetAccounts().Where(w => w.HasKey))
+                keys.Add(new()
+                {
+                    Address = account.Address,
+                    ScriptHash = account.ScriptHash,
+                    PublicKey = account.GetKey().PublicKey,
+                });
+            return Ok(keys);
+        }
+
+        /// <summary>
+        /// Change wallet password.
+        /// </summary>
+        /// <param name="sessionId" example="066843daf5ce45aba803587780998cdb">Session Id of the open/created wallet.</param>
+        /// <param name="model"></param>
+        /// <returns>Empty body response.</returns>
+        /// <response code="200">Successful</response>
+        /// <response code="400">An error occurred. See Response for details.</response>
+        [HttpPost("{session:required}/changepassword", Name = "WalletChangePassword")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        public IActionResult WalletChangePassword(
+            [FromRoute(Name = "session")]
+            Guid sessionId,
+            [FromBody]
+            WalletChangePasswordModel model)
+        {
+            if (WalletSessions.ContainsKey(sessionId) == false)
+                throw new KeyNotFoundException(sessionId.ToString("n"));
+            if (model.OldPassword == model.NewPassword)
+                throw new WalletException($"{nameof(model.OldPassword)} is the same as {nameof(model.NewPassword)}.");
+            var session = WalletSessions[sessionId];
+            var wallet = session.Wallet;
+            session.ResetExpiration();
+            if (wallet.VerifyPassword(model.OldPassword) == false)
+                throw new WalletException("Invalid password! Session terminated!");
+            if (model.CreateBackupFile && wallet is NEP6Wallet)
+            {
+                var bakFile = wallet.Path + $".{sessionId:n}.bak";
+                if (System.IO.File.Exists(wallet.Path) == false)
+                    throw new WalletException("Create wallet backup failed. Wallet file doesn\'t exist.");
+                if (System.IO.File.Exists(bakFile) && model.OverwriteIfBackupFileExists == false)
+                    throw new WalletException("Backup File already exists for this session.");
+                System.IO.File.Copy(wallet.Path, bakFile, true);
+            }
+            if (wallet.ChangePassword(model.OldPassword, model.NewPassword) == false)
+                throw new WalletException("Failed to change password!");
+            if (wallet is NEP6Wallet nep6)
+                nep6.Save();
+            return Ok();
```

Comment:
```suggestion
            return Ok("Password is changed.");
```
Make sure password is changed.

Reply:
i think http status is good indicator, because it's a `Rest` service. Normally developers use http status for this. Have a look at the top of function code comments. You will see the comments.